src/main/rx/rx.h

```diff
@@ -88,6 +88,7 @@ typedef struct rxConfig_s {
 
     uint16_t rx_min_usec;
     uint16_t rx_max_usec;
+    uint16_t rx_fail_usec[MAX_SUPPORTED_RC_CHANNEL_COUNT];
```

Comment:
ok, so this uses a bit of ram, we could use less by using uint8_t in the same way the channelModeRanges do, see this:

https://github.com/cleanflight/cleanflight/blob/master/src/main/io/rc_controls.c#L281-L289

and this

https://github.com/cleanflight/cleanflight/blob/master/src/main/io/rc_controls.h#L89-L96

Basically, instead of allowing the user to set any arbitrary value you allow them to use values in say a range between 750 and 2250 in steps of 50, then you just store the step offset in a uint8_t.

i.e. `sample = 750 + (50 * rx_fail_usec_step[rawChannel]);`